content-repo/extra-docs/articles/troubleshooting.md

```diff
@@ -133,79 +119,194 @@ Examples of common errors indicating that there is an issue with trusting a TLS/
+#### Check the fetch history
 
-    * Identify: `Endpoint certificate` -> `Subject` -> `CommonName` and `certificate` -> `Extentions` -> `SubjectAlternativeName`:
+Check the [fetch history](#fetch-history) for the following:
 
-      ![image-20201018160939173](../../../docs/doc_imgs/reference/certificate-verification-altnames.png)
+1. Temporary error from the API - Sometimes fetch mechanism can encounter temporary issues when calling the API, such issues will usually appear once or twice before getting back to fetch normally and will contain the 500 (internal server) error.
+2. Params collision - In some integrations, there are params that can’t be configured together and will throw an error if they do. The error should appear in the fetch history and should be informative about which two params can’t be configured together.
+3. Error from the API related to malformed params - Sometimes an error can come up from the API informing us that there was an issue with the params passed as part of the request, this error usually occurs in integrations where there’s a free-text filter param that might be malformed, double check all params are correct and ensure the error message doesn’t point to such issue.
+4. Error from the API, getting 429 (rate limit) error - Some APIs have a rate limiter in their system configured to a certain value that might be reached. when getting 429 try to increase the fetch interval as it will result less calls in a certain amount of time, and raise the limit as it usually affect the number of calls due to the data being fetched in less time.
 
-      ![image-20201018160950403](../../../docs/doc_imgs/reference/certificate-verification-common-name.png)
+#### Reset last run
 
-    * Resolution: If the URI isn't matching the URI endpoint (Regex), try to access the endpoint with one of the alt names/common names. If the endpoint isn't accessible via trusted names, sign the certificate with the correct common name or apply an additional alt name.
+Sometimes the fetch mechanism is getting stuck on a certain request due to last_run issues. when failing to get new events, head to the instance configuration, and press the **reset last run** button at the bottom of the page.
 
+### Getting aggregated delays / failing to get events without an error error
 
-## Fetch Incidents Troubleshooting
-### Fetch History
-In XSOAR Versions 6.8 and above, it is possible to observe the results of the last **fetch-incidents**/**fetch-indicators** runs using the Fetch History modal. To view the modal, click the button with the history icon next to the Integration Instance settings.
-<img src="../../docs/doc_imgs/incidents/fetchhistory.gif"></img>
+In case you’re having aggregated delays between the fetched events and the real time events or you suspect you're not getting new events although there's no error in the [fetch-history](#fetch-history).
```

Comment:
i dont think this is common enough that we should document. maybe under a "fetch troubleshooting advanced" collapsed section, but i dont think the akamai case is common enough that it should have its own flow in the troubleshooting docs

Reply:
I think that if you look at this section as a whole, including the the 1,2 below then it's worth keeping.